Cargo.toml

```diff
@@ -16,7 +16,6 @@ keywords = ["docker"]
 # OpenSSL is fairly hard to build on certain platforms, especially if you
 # want to produce release binaries.  So we enable it by default, but allow
 # it to be excluded using `default-features = false`.
-default = ["ssl"]
```

Comment:
It's my slippiness... I removed it because it broke my build and I forgot to put it back.